src/syntax/FStar.Syntax.Syntax.fsti

```diff
@@ -180,10 +180,69 @@ and letbinding = {  //let f : forall u1..un. M t = e
     lbattrs:list attribute; //attrs
     lbpos  :range;           //original position of 'e'
 }
-and antiquotations = list (bv * term)
+and antiquotations = int * list term
```

Comment:
Does it need a checked file version increment?

Reply:
Yes, thanks! Doing it now.